Code-4-Community/speak-for-the-trees-frontend
# Add AddSite route to frontend

## Why

[ClickUp Ticket](https://app.clickup.com/t/2mrderx)


Add an AddSite form to the frontend Admin page that allows admins and super-admins to add new sites individually.

## This PR


* Use two existing forms, `EditSiteForm` and `UpdateSiteForm`, to allow users to add a site and its site entry to the database.
* Have the `Form` component in `EditSiteForm` take up 100% of the available width.
* Dependent on https://github.com/Code-4-Community/speak-for-the-trees-backend-v2/pull/110

## Screenshots


Admin dashboard (desktop + Firefox): 
![image](https://user-images.githubusercontent.com/33704204/162588356-0e90cb25-878f-47f8-9328-fa8c6bd8dc45.png)

## Verification Steps

 
Visually verified that all components on the admin page and the `UpdateSiteForm` component on the site page were rendered as expected using Firefox and Chrome (on desktop and mobile). Attempted to add sites with invalid and missing field values to ensure that errors were thrown when invalid inputs are given and that the site is correctly created otherwise.
